ee/tabby-webserver/src/service/cron/mod.rs

```diff
@@ -16,29 +16,38 @@ async fn new_job_scheduler(jobs: Vec<Job>) -> anyhow::Result<JobScheduler> {
     Ok(scheduler)
 }
 
-pub fn run_cron(db_conn: &DbConn) {
+pub fn run_cron(db_conn: &DbConn, turn_on_scheduler_jobs: bool) {
     let db_conn = db_conn.clone();
     tokio::spawn(async move {
+        let mut jobs = vec![];
+
         let Ok(job1) = db::refresh_token_job(db_conn.clone()).await else {
             error!("failed to create db job");
             return;
         };
-        // run every 5 minutes
-        let Ok(job2) =
-            job_utils::run_job(db_conn.clone(), "sync".to_owned(), "0 1/5 * * * * *").await
-        else {
-            error!("failed to create sync job");
-            return;
-        };
-        // run every 5 hours
-        let Ok(job3) =
-            job_utils::run_job(db_conn.clone(), "index".to_owned(), "0 0 1/5 * * * *").await
-        else {
-            error!("failed to create index job");
-            return;
-        };
+        jobs.push(job1);
+
+        if (turn_on_scheduler_jobs) {
```

Comment:
Looks fine for now, but if we end up adding more feature-gated scheduled jobs, we might want to try to take a more declarative approach to this

Reply:
sure - a env flag / config flag based approach could be adopt in future